src/main/java/bton/ci536/fizzit/trade/LocalTradeList.java

```diff
@@ -30,6 +31,8 @@
     private double totalValue = 0;
     private int totalItems = 0;
     
+    private Customer customer;

```

Comment:
Why do you need Customer here?

Reply:
Oh that was for something I was going to attempt. But it's not needed now. Will get rid of when I get home.